README.md

```diff
@@ -24,12 +24,6 @@ It is super easy to use App Center. Have a look at our [get started documentatio
 
 We are looking forward to your contributions via pull requests.
 
```

Comment:
We can add a relative link to the new markdown file here

Reply:
We don't have such links for other platforms, so let's keep it harmonized. "CONTRIBUTING" is a special filename, so GitHub should add link to it when someone opens a pull request or creates an issue. See   https://docs.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors